src/include/stir/recon_buildblock/ProjMatrixElemsForOneBin.h

```diff
@@ -113,6 +115,8 @@ class ProjMatrixElemsForOneBin
   inline Bin get_bin() const;
   //! and set the bin coordinates
   inline void set_bin(const Bin&);
+  //! get a ref to the bin
+  inline Bin* get_bin_ptr();
```

Comment:
cannot retrun a bare pointer. I see this is then because `get_bin` returns a copy, but this is too dangerous.

We could replace `get_bin` to return a `const Bin&`. I see no harm in that.